CMakeLists.txt

```diff
@@ -1595,19 +1595,22 @@ endif()
 
 if(WITH_INF_ENGINE OR INF_ENGINE_TARGET)
   if(INF_ENGINE_TARGET)
+    list(GET INF_ENGINE_TARGET 0 ie_target)
```

Comment:
Why is these changes here? They should be on the 3.4 branch already: #15376

Please rebase patch.